### DISCUSSION TITLE
Warning when trying to install mdb tw-elements of exceeded listeners asked to change emitter.setMaxListener()

### DISCUSSION THREAD

**Discussion by nikkimarie31** [mdbootstrap/TW-Elements#1969] [Support from community]
So i was following along with the tutorial for mdb tw-elements and when i typed in npm install and then got the warning possible EventEmitter memory leak detected and my guess is setting the limit higher to unlimited is not the answer. may someone please guide me in the correct direction. i already read the documents but the only thing that helped with is how to set it to unlimited thank you 

i couldnt find any other articles on this on here

**Comment by juujisai** [comment]
Hi! I have few questions that could helps us.

Did you install the package in vanilla JS? Which guide did you use and after what step the issue occured? Was the project a fresh install or existing one?  Did you check if in any other browser the issue was still there?
